deepmd/loss/ener.py

```diff
@@ -268,6 +268,339 @@ def eval(self, sess, feed_dict, natoms):
         return results
 
 
+class EnerSpinLoss(Loss):
```

Comment:
This class is not tested

Reply:
Could you give me a test example about `EnerStdLoss `？

Reply:
You can use dp train to train a model then it will be covered